packages/pwa-buildpack/src/WebpackTools/PWADevServer.js

```diff
@@ -44,8 +44,8 @@ const PWADevServer = {
             contentBase: false, // UpwardPlugin serves static files
             compress: true,
             hot: true,
-            host: '0.0.0.0',
-            port: await portscanner.findAPortNotInUse(10000),
+            host: config.host || '0.0.0.0',
+            port: config.port || (await portscanner.findAPortNotInUse(10000)),
```

Comment:
maybe these values should just be checked against `process.env` values rather than messing with them in [webpack.config.js](https://github.com/magento-research/pwa-studio/pull/859/files#diff-1acbde853507ec5d5dac59ef5301a593R176)